release-deployment.md

```diff
@@ -138,20 +138,43 @@ There's nothing special about that. Each developer follows the above [Develop a
 1. Finish off the tasks in the release checklist. Once everything is done, close
    the release PR.
 
-**TBD: Discuss**
-Mike N: Long-lived release branches, yes/no?
-
-1. Create the release on Github from `master`.
+1. Delete the release branch on Github.
 
 ### Change in plan, pull a feature from a release
 
 **TBD: Discuss**
 Mike N: That probably means recreating the release branch, unless we have short-lived release branches
 
-### Change request
+### Supporting old releases
+
+In a release based project old releases can often need some maintenance. Critical bug fixes and strategic feature requests need to be supported on old releases. Old releases often become incompatible with the most recent versions of projects.
+
+In order to support old release versions gitflow has introduced the concept of support branches. Support branches are long living branches created to support major or minor versions of the project. Support branches do not get merged back into `master` or `develop` (this would cause major merge issues which are time consuming and error prone if attempted). Instead commits can be cherry picked from the support branch back into `develop`. 
+
+Support branches can be thought of as the `master` branch for old releases. Support branches for major releases should be named as `support-v<major>.x`. Support branches for minor releases should be named as `support-v<major>.<minor>.x`.
+
+Here is an example of creating a support branch for v1.0 assuming v2.0 of the project has already been released.
+
+1. Create the support branch and release branch for the patch release.
+
+    ```
+    git checkout v1.0.0
+    git checkout -b support-v1.x
+    git checkout -b release-v1.0.1
+    ```
+
+1. Make changes in the `release-v1.0.1`. Multiple PRs can be merged into this branch if several changes are necessary.
+
+1. Create release PR to merge `release-v1.0.1` into `support-v1.x`.
+
+1. Follow the standard release process treating `support-v1.x` as the `master` branch.
+
+1. Mark `support-v1.x` as a protected branch in github so that it does not get accidentally deleted.
+
+1. As PRs are merged into the `release-v1.0.1` branch create associated PRs that cherry pick the changes back into `develop`. Ensure that these changes are desired by the team going forward and that they are compatible with the current state of the `develop` branch.
```

Comment:
This is a bit confusing... at this point haven't we already merged `release-v1.0.1` back into `support-v1.x`?

Reply:
You can cherry pick the changes back into `develop` as you go or when you are finished the release. Since you are cherry picking you can do it whenever you like but I would suggest the earlier the better so that you don't forget to do it. I will move the item higher in the ordered list so that it makes more sense.